Browsable FAQs/QA/flows/Browsable FAQs.md

```diff
@@ -297,7 +298,7 @@ card DisplayContent when isnumber(content_data.body.body.text.value.image), then
     get(
       "https://content-repo-api-qa.prk-k8s.prd-p6t.org/api/v2/images/@image_id/",
       headers: [
-        ["Authorization", "Token @globals.config.contentrepo_token"]
+        ["Authorization", "Token @global.config.contentrepo_token"]
```

Comment:
I'm not sure how or when this was fixed, but I got this fix when flow wrangler pulled from Turn